frontend/packages/core/src/Theme/types.tsx

```diff
@@ -41,3 +41,12 @@ export interface ClutchColors {
 }
 
 export type ComponentState = "hover" | "focused" | "pressed" | "selected" | "disabled";
+
+export enum ThemeSpacing {
+  None = 0,
+  ExtraSmall = 0.5,
+  Small = 1,
```

Comment:
I assume the numbers would be like `0.5 * 8`?

Reply:
That's right. The `spacing` value set in the theme is the scaling factor. 

At first I was trying to document each member with the resulting value, like `4px` , but it would be outdated if the spacing value changes in the future.